taxcrunch/tests/test_multi_cruncher.py

```diff
@@ -0,0 +1,52 @@
+import pytest
+import json
+import os
+import numpy as np
+import pandas as pd
+import taxcalc as tc
+import taxcrunch.cruncher as cr
+import taxcrunch.multi_cruncher as mcr
+
+def test_create_data():
```

Comment:
To streamline things a bit, you could remove this test and add `assert isinstance(b, mcr.Batch)` to the `test_get_pol_no_reform()` test